includes/admin/helper/class-wc-helper.php

```diff
@@ -1098,6 +1098,18 @@ private static function _get_local_from_product_id( $product_id ) {
 		return false;
 	}
 
+	/**
+	 * Checks whether current site has product subscription of a given ID.
+	 *
+	 * @param int $product_id The product id.
+	 *
```

Comment:
Could you please add a @ since tag here?

Reply:
Added in https://github.com/woocommerce/woocommerce/pull/24123/commits/f51265e58db0a6eb06d694bb4f1766c6890a6d78.